### ISSUE TITLE
Recommend using v2 instead of v2…

### ISSUE THREAD

**Issue by tmcw** [tmcw/notfoundbot#39]
Do GitHub actions use semver notation? If so, I should probably recommend it so that we can ship auto-updates.

**Comment by icco** [comment]
https://github.community/t/versioning-guidance-for-authoring-and-consuming-actions/138763/2 talks a bit about this

**Comment by bensheldon** [comment]
Yep. Dependabot has a package-ecosystem for GitHub Actions that will propose version bumps based on semantic versioning rules. It's very helpful.

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs
